src/main/java/frc/robot/subsystems/SwerveDrivetrain.java

```diff
@@ -62,10 +92,20 @@ public void periodic() {
             SmartDashboard.putNumber("FR_ABS",frontRight.getSwerveMotor().getAbsEnc().getAbsolutePosition());
             SmartDashboard.putNumber("BL_ABS",backLeft.getSwerveMotor().getAbsEnc().getAbsolutePosition());
             SmartDashboard.putNumber("BR_ABS",backRight.getSwerveMotor().getAbsEnc().getAbsolutePosition());
+            SmartShuffleboard.put("DriveTrain", "Front Left Drive Current", frontLeftDriveCurrent);
```

Comment:
why do we need all these on the dashboard? maybe the total drive and total steer, but not the individual values.